tz.go

```diff
@@ -49,13 +49,19 @@ func main() {
 	}
 
 	// set 24
-	block := 7
+	block := 8
```

Comment:
Why increasing by just one here?

Reply:
I tested this a good amount on a lot of different terminal sizes and with various numbers both higher and lower. `8` just seemed to be the best one while still using up almost all of the terminal window.